pkgs/development/tools/misc/coccinelle/default.nix

```
@@ -34,7 +49,7 @@ stdenv.mkDerivation rec {
     '';
 
     homepage = "http://coccinelle.lip6.fr/";
-    license = lib.licenses.gpl2;
+    license = lib.licenses.gpl2Plus;
```

`coccinelle` does not allow distribution under the terms of a later GPL version.

```suggestion
    license = lib.licenses.gpl2Only;
```